From 6a1b2740a809055c7cc0c9dfaff4dd10027839fb Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Fri, 18 Jan 2013 16:30:40 +0800
Subject: ENGR00240740-2 ARM:IPUv3:Add an interface to disable IPU hsp clk

This patch adds an interface to disable IPU hsp clock so that it
can be called out of ipu common driver.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit 78f0495e79f3b18d3f56ac8bf6ca76a5cea91cf5)
---
 arch/arm/plat-mxc/include/mach/ipu-v3.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/plat-mxc/include/mach/ipu-v3.h b/arch/arm/plat-mxc/include/mach/ipu-v3.h
index fb9dc38..d5ee6af 100755
--- a/arch/arm/plat-mxc/include/mach/ipu-v3.h
+++ b/arch/arm/plat-mxc/include/mach/ipu-v3.h
@@ -587,6 +587,7 @@ struct ipu_soc;
 struct ipu_soc *ipu_get_soc(int id);
 int32_t ipu_init_channel(struct ipu_soc *ipu, ipu_channel_t channel, ipu_channel_params_t *params);
 void ipu_uninit_channel(struct ipu_soc *ipu, ipu_channel_t channel);
+void ipu_disable_hsp_clk(struct ipu_soc *ipu);
 
 static inline bool ipu_can_rotate_in_place(ipu_rotate_mode_t rot)
 {
-- 
1.7.9.5

